samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -30,6 +40,30 @@ BUILD_ASSERT(IS_ENABLED(CONFIG_SCAN_SELF) || IS_ENABLED(CONFIG_SCAN_OFFLOAD),
 #define PA_SYNC_SKIP              5
 #define NAME_LEN                  sizeof(CONFIG_TARGET_BROADCAST_NAME) + 1
 
+
+
+#if defined(CONFIG_LIBLC3)
+#define MAX_SAMPLE_RATE         (48000U)
+#define MAX_FRAME_DURATION_US   (10000U)
+#define MAX_NUM_SAMPLES_MONO    ((MAX_FRAME_DURATION_US * MAX_SAMPLE_RATE) / USEC_PER_SEC)
+#define MAX_NUM_SAMPLES_STEREO  (MAX_NUM_SAMPLES_MONO * 2)
+
+#define LC3_ENCODER_STACK_SIZE  4096
+#define LC3_ENCODER_PRIORITY    5
+#endif /* defined(CONFIG_LIBLC3) */
+
+#if defined(CONFIG_USB_DEVICE_AUDIO)
+#define USB_SAMPLE_RATE	             (48000U)
+#define USB_FRAME_DURATION_US        (1000U)
+#define USB_TX_BUF_NUM               (10U)
+#define BROADCAST_MONO_SAMPLE_SIZE   (MAX_NUM_SAMPLES_MONO * sizeof(int16_t))
+#define BROADCAST_STEREO_SAMPLE_SIZE (BROADCAST_MONO_SAMPLE_SIZE * sizeof(int16_t))
+#define USB_STEREO_SAMPLE_SIZE       ((USB_FRAME_DURATION_US * USB_SAMPLE_RATE * sizeof(int16_t) \
+				      * 2) / USEC_PER_SEC)
```

Comment:
The `sizeof(int16_t)` is because the data is int16_t? 

Adding a comment or a new macro to denote that might be useful